redis-module/src/main/java/team/asd/redis/RedisPoolManager.java

```diff
@@ -0,0 +1,10 @@
+package team.asd.redis;
+
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+
+public class RedisPoolManager {
+	public static JedisPool getPool() {
```

Comment:
The current implementation creates a new JedisPool each time getPool() is called, which can lead to resource exhaustion. Try to use Singleton pattern to ensure only one instance of JedisPool is created and reused.